mobify/platform-scaffold
# Fix product variants for item in cart incorrect

# Product variants for item in cart incorrect

Previously, every cart item would show `Maroon, XL` for color and size. This PR shows the correct selected variant in the cart.

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-91

## Changes
- Added selected variant options to the cart product list view

## How to test-drive this PR
- Use SFCC connector
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add some item to cart with variants selected
- Add another item with different variants
- Ensure that the cart is showing the correct variants
